universal-login-commons/src/core/constants/test.ts

```diff
@@ -40,7 +41,7 @@ export const TEST_EXECUTION_OPTIONS = {gasPrice: TEST_GAS_PRICE, gasLimit: TEST_
 export const TEST_APPLICATION_INFO = {
   applicationName: 'UniversalLogin',
   logo: 'logo',
-  type: 'laptop',
+  type: 'laptop' as DeviceType,
 };
```

Comment:
I think it might be better write something like this: 
```ts
export const TEST_APPLICATION_INFO: ApplicationInfo = {
  applicationName: 'UniversalLogin',
  logo: 'logo',
  type: 'laptop' as DeviceType,
}
```